crates/store/re_grpc_server/README.md

```diff
@@ -8,3 +8,13 @@ Part of the [`rerun`](https://github.com/rerun-io/rerun) family of crates.
 ![Apache](https://img.shields.io/badge/license-Apache-blue.svg)
 
 Server implementation of an in-memory Storage Node.
+
+## Usage
+
+To run the server locally:
+
+1. Install [Envoy](https://www.envoyproxy.io/docs/envoy/latest/start/install)
+2. `cargo run -p re_grpc_server --release`
+3. `envoy -c envoy.yml`
+
+The server is available on `127.0.0.1:1853`.
```

Comment:
This is different from the port that is printed by crates/store/re_grpc_server/src/main.rs

But it's better to just remove this and rely on `cargo run -p re_grpc_server` printing out the correct port. Less risk of code rot

```suggestion
```